pkg/descheduler/strategies/duplicates.go

```diff
@@ -36,6 +36,8 @@ import (
 	"sigs.k8s.io/descheduler/pkg/utils"
 )
 
+const RemoveDuplicatesStrategyName = "RemoveDuplicates"
```

Comment:
```suggestion
const RemoveDuplicatesStrategyName = "RemoveDuplicatePods"
```

I don't love the name 🤷🏼 but in spirit of keeping it the same: https://github.com/kubernetes-sigs/descheduler/blob/v0.24.0/pkg/descheduler/strategies/duplicates.go#L213